### DISCUSSION TITLE
Welcome to Shadertoys Discussions!

### DISCUSSION THREAD

**Discussion by nmbr73** [nmbr73/Shadertoys#20] [General]
## Here we go...

Shader Languages, DCTL, Fusion, Lua/Python/C/C++, .. there is so much to explore. We are using Discussions as a place to connect and hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other people interested in the subject matter
  * Welcome others and are open-minded

Remember that this is a space we build together.

To get started, comment below with an introduction of yourself and tell us about your contribution and your expectations.

**Comment by rne1223** [comment]
I like the Wiki. It should cut down the learning curve.
